templates/hooks--fsmonitor-watchman.sample

```diff
@@ -63,8 +64,7 @@ sub launch_watchman {
 	my $query = <<"	END";
 		["query", "$git_work_tree", {
 			"since": $time,
-			"fields": ["name"],
-			"expression": ["not", ["allof", ["since", $time, "cclock"], ["not", "exists"]]]
+			"fields": ["name"]
```

Comment:
Same comment: Update the above comment to better match the query we're performing here.